libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/impl/VideoCompressorConfig.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.libraries.mediaupload.impl
+
+import androidx.annotation.OptIn
+import androidx.media3.common.util.Size
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.transformer.VideoEncoderSettings
+import kotlin.math.min
+import kotlin.math.roundToInt
+
+@OptIn(UnstableApi::class)
+internal object VideoCompressorConfigFactory {
+    // 720p
+    private const val MAX_COMPRESSED_PIXEL_SIZE = 1280
+
+    // 1080p
+    private const val MAX_PIXEL_SIZE = 1920
```

Comment:
Update (/remove) those 2 comments?

Reply:
I think they're fine, we're comparing the 'major' dimension of those 2 resolutions. I can add more details in the comments.

Reply:
Ah OK, so it's 720*1280 and 1080*1920. Maybe updating the comment is good yes, thanks!